docs/reference/glossary.rst

```diff
@@ -64,6 +64,10 @@ Glossary
     Leveled compaction strategy (LCS)
       :abbr:`LCS (Leveled compaction strategy)` uses small, fixed-size (by default 160 MB) SSTables divided into different levels. See :doc:`Compaction Strategies</architecture/compaction/compaction-strategies/>`.
 
+    Liveness
+      The ability to update a configuration property without restarting the node. Properties that support live updates can be updated via the ``system.config`` virtual table or the REST API.
```

Comment:
please note, we have at least three ways to update the `LiveUpdate` options:

1. update the config file, send `SIGHUP`
2. `system.config` virtual table
3. RESTful API

we only mention the most used way here. also, would be better if we can have a link to `operating-scylla/system-configuration/` ?